features/step_definitions.rb

```diff
@@ -1,12 +1,13 @@
 Before do
+  FileUtils.rm_rf(Paths.test_dir) if Paths.test_dir.exist?
   FileUtils.mkdir_p(Paths.test_dir) unless Paths.test_dir.directory?
   Dir.chdir(Paths.test_dir)
 end
 
 #
 
 After do
-  Paths.test_dir.rmtree if Paths.test_dir.exist?
```

Comment:
`.rmtree` does not fail per se. The feature directories do get removed completely (verified locally), but on Windows it also raises an alarm:
```
  Permission denied @ dir_s_rmdir - C:/projects/jekyll/tmp/jekyll (Errno::EACCES)
  C:/projects/jekyll/features/step_definitions.rb:9:in `After'
```
I stumbled upon the fix when I tried the *identical*  `FileUtils.rm_rf` method. This method manipulates files rather than try to modify the directory attribute.